services/sharing/pkg/revaconfig/config.go

```diff
@@ -104,11 +103,12 @@ func SharingConfigFromStruct(cfg *config.Config) map[string]interface{} {
 							"machine_auth_apikey": cfg.PublicSharingDrivers.CS3.SystemUserAPIKey,
 						},
 						"jsoncs3": map[string]interface{}{
-							"gateway_addr":        cfg.Reva.Address,
-							"provider_addr":       cfg.PublicSharingDrivers.JSONCS3.ProviderAddr,
-							"service_user_id":     cfg.PublicSharingDrivers.JSONCS3.SystemUserID,
-							"service_user_idp":    cfg.PublicSharingDrivers.JSONCS3.SystemUserIDP,
-							"machine_auth_apikey": cfg.PublicSharingDrivers.JSONCS3.SystemUserAPIKey,
+							"gateway_addr":                       cfg.Reva.Address,
+							"provider_addr":                      cfg.PublicSharingDrivers.JSONCS3.ProviderAddr,
+							"service_user_id":                    cfg.PublicSharingDrivers.JSONCS3.SystemUserID,
+							"service_user_idp":                   cfg.PublicSharingDrivers.JSONCS3.SystemUserIDP,
+							"machine_auth_apikey":                cfg.PublicSharingDrivers.JSONCS3.SystemUserAPIKey,
+							"writeable_share_must_have_password": cfg.WriteableShareMustHavePassword,
```

Comment:
Does only `jsoncs3` driver support enforce password support? Otherwise we should add it to other drivers also.

Reply:
Yes, at the moment only the jsoncs3 driver supports it.